src/directory/__tests__/getMetaStringObj.test.js

```diff
@@ -0,0 +1,39 @@
+import { getMetaStringObj } from '../getMetaStringObj.mjs';
+import JSON5 from 'json5';
+
+describe('getMetaStringObj', () => {
+  it('should return meta object for a valid page file path with valid meta object', async () => {
+    const filePath = './src/directory/__tests__/helpers/valid-meta.mdx';
+    const expectedMetaObject = JSON5.parse(
```

Comment:
It seems like I couldn't get the object compare to work unless I parsed the expected object using JSON5. I guess that makes sense since we use that to parse the result in getMetaStringObj?

Reply:
What did the `expectedMetaObject` look like before you tried using JSON5?

Reply:
Updated the latest to use the not-parsed `expectedMetaObject` because after re-running that seems to work now?